src/CodeGenerator/Generator/GeneratorHelper.php

```diff
@@ -12,6 +12,7 @@
  * Small methods that might be useful when generating code.
  *
  * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ * @author Jérémy Derussé <jeremy@derusse.com>
```

Comment:
thanks :hugs:

Reply:
We should remove all or slap your name all over the place.